udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,10 +90,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /**
-   * Obtains parameter from Comments typing field.
+   * Gets count for comments to be shown.
    */
+  private int getCommentCount(HttpServletRequest request) {
+    // Get the input from the form.
+    String commentCountString = request.getParameter("comment-count");
+
+    // Convert the input to an int.
+    int commentChoice;
```

Comment:
`commentChoice` sounds like it could be a string (it's the chosen comment).

I'd suggest `commentCount` instead, as that's more clearly a number, and also better shows that this number represents a limit.

Reply:
Done